imagemounter/_util.py

```diff
@@ -130,6 +134,18 @@ def check_output_(cmd, *args, **kwargs):
         raise
 
 
+def get_free_nbd_device():
+    try:
+        result = check_output_(['/bin/bash', '-c', 'for x in /sys/class/block/nbd*; do S=`cat $x/size`; '
+                                                 'if [ "$S" == "0" ]; then echo "/dev/`basename $x`"; '
+                                                 'break; fi; done']).strip()
```

Comment:
Wouldn't it be better to do this in Python code instead? Something like the extremely untested:

```python
for nbd_path in glob.glob("/sys/class/block/nbd*"):
    if check_output_("cat " + nbd_path + "/size") == "0":
        return "/dev/" + os.basename(nbd_path)
```